packages/cli/src/commands/generate/helpers.js

```diff
@@ -170,3 +171,22 @@ export const intForeignKeysForModel = (model) => {
     .filter((f) => f.name.match(/Id$/) && f.type === 'Int')
     .map((f) => f.name)
 }
+
+export const isWordNonPluralizable = (word) => {
+  return pluralize.isPlural(word) === pluralize.isSingular(word)
+}
+
+/**
+ * @description Adds an s, if it it can't pluralize the word
+ */
+export const forcePluralizeWord = (word) => {
+  // If word is already plural, check if plural === singular, then add s
+  // else use plural
+  const shouldAddS = isWordNonPluralizable(word) // equipment === equipment
+
+  if (shouldAddS) {
+    return `${word}s`
```

Comment:
Is it worth it to add "es" instead of just "s" if the word already ends in "s"?
So we get for example "chassises" instead of "chassiss" or "serieses" instead of "seriess"

Or go with DT's suggestion of prefixing "many".
"chassis" -> "manyChassis", "series" -> "manySeries"

Reply:
Yep this is a great idea. Let me add this in, after lunch

Reply:
I'd vote for prefixing with manySeries over serieses

Reply:
Done